source/_references/searching.md

```diff
@@ -0,0 +1,155 @@
+
+var_dump($sourceImages);
+```
+
+The response will contain an array of objects that look like a single image.
+
+## Paging the results
+
+You can limit the outputs of the list by using limit and offset parameters to page through them.
+
+| Attribute | Description | Default |
+| -------------- | ------------- | ------------- |
+| limit | Optional limit | 100 |
+| offset | Optional offset | 0 |
+
+```bash
+curl -X GET 'https://api.rokka.io/sourceimages/testorganization?offset=100limit=20'
+```
+```php
+$client = \Rokka\Client\Factory::getImageClient('testorganization', 'apiKey', 'apiSecret');
+
+$sourceImages = $client->listSourceImages(20, 100);
+
+var_dump($sourceImages);
+```
+
+### Deep Paging Beyond 10,000 Hits
+
+If you want to page beyond 10'000 hits, you need to use a so-called _cursor_. With a cursor you can only page forward by one "page" and not randomly access a page.
+You can use this cursor value as input for the _offset_ parameter to page to the next page. A full correct link is also provided in the resulting JSON.
+If you don't submit an _offset_ number and your result has more than 10'000 hits, it will automatically return a link for the next page with the cursor value in it.
+But you can still use offset numbers (instead of cursor) for paging up to 10'000 hits.
+
+A typical resultset with more than 10'000 hits may look like the following
+
+```bash
+curl -H 'Content-Type: application/json' -X GET 'https://api.rokka.io/sourceimages/testorganization?limit=1'
+```
+```json
+{
+   "items" : [
+      {
+         "hash" : "abf3d6a12c5532612b7109d54abde391505b3669",
+         "binary_hash" : "106ea5d2d1a73f586455effa371054ab15792a40",
+         "width" : 699,
+         "link" : "/sourceimages/testorganization/abf3d6a12c5532612b7109d54abde391505b3669",
+         "height" : 1200,
+         "size" : 826921,
+         "organization" : "testorganization",
+         "created" : "2017-01-18T08:31:25+0000",
+         "format" : "png",
+         "name" : "foobar.png"
+      }
+   ],
+   "total" : 239208,
+   "links" : {
+      "next" : {
+         "href" : "/sourceimages/testorganization?limit=1&offset=VX3_3FBUQW9KNDlQYXMyTmtDUHhOd2NtOWtOR0ptTTJRMllURXlZekl5TXpNM01USmlOekV3T1dRMU5HRmlaR1V6T1RFMU1EVmlNelkyT1cxcFozSnZjdz09AQ"
+      }
+   },
+   "cursor" : "VX3_3FBUQW9KNDlQYXMyTmtDUHhOd2NtOWtOR0ptTTJRMllURXlZekl5TXpNM01USmlOekV3T1dRMU5HRmlaR1V6T1RFMU1EVmlNelkyT1cxcFozSnZjdz09AQ"
+}
+```
+
+## Filtering for particular images
+
+You can filter the list by the fields returned by a sourceimage (and its user defined metadata). Just append the fieldname and value to your list request. The following returns all images with height = 1200.
+
+
+```bash
+curl -H 'Content-Type: application/json' -X GET 'https://api.rokka.io/sourceimages/testorganization?height=1200'
+```
+
+You can also filter for user defined metadata, if you prefix the fieldname with `user:`. Don't forgot to include the type prefix ([described here](user-metadata.html)). Example:
```

Comment:
`s/filter for user defined metadata/filter by user defined metadata/`

`s/forgot/forget/`

Is the link to user-metadata.html correct? I think I've seen other links using `/documentation/references/operations.html#crop` (i.e. an absolute path). 
I'd also tend to do a bigger link text than `described here`. 

Example:
> Don't forget to include [the type prefix of the field](user-metadata.html). Example: